packages/desktop-client/src/components/schedules/SchedulesTable.js

```diff
@@ -99,7 +104,7 @@ export function ScheduleAmountCell({ amount, op }) {
             lineHeight: '1em',
             marginRight: 10
           }}
-          title={(isApprox ? 'Approximately ' : '') + str}
+          title={`${t('general.approximately')} ` + str}
```

Comment:
Since languages have different word order, this should probably be formatted as `t('general.approximately', { amount: str })`, so translations could control whether words appear before or after the amount.